src/odo/odoWrapper.ts

```diff
@@ -109,6 +109,7 @@ export class Odo {
         starter: string = undefined,
         useExistingDevfile = false,
         customDevfilePath = '',
+        devfileVersion?: string
```

Comment:
how does it know which version of devfile it should not update to? we need to define it somewhere.

Reply:
This is only a workaround to make the test working with current officially released 'odo' version  and the Default Devfile Registry.

The officially released 'odo' supports devfile schema versions `<= 2.2.0`. So this version is provided as the value of this argument hereafter in the test when creating a component from 'node.js' devfile (because this devfile has multiple versions and [if not specifying this argument] odo returns the default one, which currently is set to be `> 2.2.0.`

We do not provide this argument value when creating a component with 'Go` devfile - its devfile returned by default has  schema version `< 2.2.0` so it perfectly works with odo.